tasks/main.yml

```diff
@@ -1,15 +1,21 @@
 ---
 
-- name: "Add epel-release repository"
+- name: "Install EPEL repository package (RHEL)"
+  yum:
+    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    state: "present"
+  when: ansible_distribution == 'RedHat'
+
+- name: "Add epel-release repository (CentOS/RHEL)"
   yum:
     name: "epel-release"
     state: "latest"
-  when: ansible_distribution == 'CentOS'
+  when: ansible_os_family == 'RedHat'
 
```

Comment:
In the case of Red Hat, do you still need to do "yum install epel-release" after installing the rpm, or is this task only needed for CentOS?

Reply:
Thanks Hector, this second task is redundant for RHEL, so I think it is better to use the old conditional (CentOS only). I think I can change these task names too, for example:
 
- Add epel-release repository (RHEL)
- Add epel-release repository (CentOS)

Reply:
thanks! 👍